packages/core/src/baseResourceApi.ts

```diff
@@ -174,8 +175,14 @@ export abstract class BaseResourceAPI<ResponseType> {
     );
   }
 
-  protected async searchEndpoint<FilterType>(query: FilterType) {
-    return this.callEndpointWithTransform(query, this.callSearchEndpoint);
+  protected async searchEndpoint<FilterType>(
+    query: FilterType,
+    path?: string,
```

Comment:
can you set the default value to `this.callSearchEndpoint` to avoid breaking a lot of endpoints.

Reply:
though it seems perhaps the callEndpointWithTransform uses the default value.

Reply:
@andersfylling the support for path & queryParams is added in an optional manner.
Not sure if this breaks any end point.
Can you please help me with what change are you looking for exactly ?
callEndpointWithTransform doesn't have any default value and is not touched in this PR.

Reply:
looks good!